cl/corpus_importer/management/commands/harvard_merge.py

```diff
@@ -958,6 +994,10 @@ def handle(self, *args, **options) -> None:
                         f"Cluster id: {cluster_ids[0][0]} already merged."
                     )
                     return
+            else:
+                logger.info(
+                    f"Cluster ID: {options['cluster_id']} doesn't exist"
+                )
```

Comment:
i added it in case we wanted to merge a specific cluster but it did not exist (it happened to me sometimes in the local environment, so I added it to know what was happening)

Reply:
weird